.prettierrc

```diff
@@ -0,0 +1,23 @@
+{
+  "printWidth": 100,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": true,
```

Comment:
I think we prefer spaces over tabs for consistency reasons.

Reply:
There we go. Fixed.